powerhome/playbook
# [PLAY-35] Add Spline example to Bar Graph docs

#### Screens

##### React Example

<img width="1159" alt="react_example" src="https://user-images.githubusercontent.com/17501514/142214249-ec6b2b40-90fd-4b41-9b38-2be79703039f.png">


##### React Code Snippet

<img width="1153" alt="React_snippet" src="https://user-images.githubusercontent.com/17501514/142214353-09b70a9e-317a-46da-94c2-d02d257071b3.png">


##### Rails Example

<img width="1155" alt="Rails_example" src="https://user-images.githubusercontent.com/17501514/142214480-e1af0737-1fc0-41d1-9172-a03fd64c8f6f.png">


##### Rails Code Snippet

<img width="1149" alt="Rails_snippet" src="https://user-images.githubusercontent.com/17501514/142214772-1c6bdf58-5086-4114-bfff-72720b4e9940.png">


#### Breaking Changes

No. This PR adds an example for an already existing, but undocumented, feature of bar graphs.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-35

#### How to test this

- Navigate to /kits/bar_graph/react
- Verify the example appears in the react bar graph examples
- Verify the example appears in the rails bar graph examples

#### Checklist:
